### PR TITLE
add d3-dsv v0.3.2 with npm autoupdate config

### LINKED ISSUE #7904: [Request] Add d3-dsv
**Library name:** d3-dsv
**Git repository url:** https://github.com/d3/d3-dsv
**npm package url(optional):** d3-dsv
**License(s):** https://github.com/d3/d3-dsv/blob/master/LICENSE
**Official homepage:** http://d3js.org/
**Wanna say something? Leave message here:** useful in case you need to process csv files but do not want entire d3

### PR DESCRIPTION
**Pull request by dannyxx001**
PR for #7904 
@x09326 please help me review this PR, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/d3/d3-dsv
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/d3-dsv
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 82
   - Count of watchers: 6
   - Count of forks: 10
 * NPM download stats (optional):
   - Downloads in the last day: 266
   - Downloads in the last week: 1174
   - Downloads in the last month: 4686

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7904, cc @mbostock

### CODE REVIEW

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

File: ajax/libs/d3-dsv/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/d3/d3-dsv.git"
+  },
+  "npmName": "d3-dsv",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "d3-dsv*"
```
Comment (PeterDaveHello) [inline ajax/libs/d3-dsv/package.json]: I prefer something like `"**/!(package.js)"`

**Comment by dannyxx001** [comment]
Sorry for late update. @x09326 I have updated it. Please help me review again, thank you.

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

**Pull request merged**